styles.css

```diff
@@ -1,3 +1,793 @@
+    gap: 8px;
+    font-size: 16px;
+    color: #000;
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-weight: 400;
+    line-height: 24px;
+    padding: 12px 24px;
+    border: none;
+    height: 48px;
+    width: 149px;
+    background: #21E786;
+    cursor: pointer;
+}
+
+.breadcrumbs-nav a {
+    font-size: 14px;
+    color: #888B8E;
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-weight: 400;
+    line-height: 20px;
+}
+
+.main-heading,
+.heading {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-weight: 400;
+    font-size: 44px;
+    line-height: 56px;
+    color: #fff;
+    margin: 0;
+}
+
+.sub-heading,
+.how-to-text {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 32px;
+    color: #fff;
+}
+
+.how-to-heading {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 22px;
+    font-weight: 400;
+    line-height: 30px;
+    text-align: center;
+    color: #21E786;
+
+}
+
+.ot-member-pos,
+.faq-description,
+.joc-text,
+.sub-text {
+    font-family: Poppins, Arial, sans-serif;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 28px;
+    color: #C2C3C5;
+}
+
+.copyright-text {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+
+}
+
+.visually-hidden {
+    visibility: hidden;
+}
+
+.nft-card,
+.about-us-list-content,
+.step-list,
+.roadmap-phase-list,
+.roadmap-nav-list,
+.faqs-list,
+.social-links-logo,
+.social-links,
+.filter-list,
+.catalog-list,
+.blog-list,
+.tag-list,
+.social-list,
+.categories-list,
+.recent-post-list {
+    list-style: none;
+}
+
+.list-term {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 28px;
+    color: #C2C3C5;
+}
+
+.list-description {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 56px;
+    font-weight: 400;
+    line-height: 68px;
+    color: #fff;
+    text-shadow: 0 4px 16px #FFFFFF66;
+}
+
+.list-item-number {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-weight: 400;
+    font-size: 80px;
+    line-height: 96px;
+    color: #21E786;
+    opacity: 30%;
+}
+
+.list-item-sub {
+    font-family: Poppins, Arial, sans-serif;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 28px;
+    text-align: center;
+    color: #C2C3C5;
+}
+
+.joc-text-diff,
+.sub-info {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 28px;
+    text-align: center;
+    color: #F7F7F9;
+}
+
+.page-footer-text {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 26px;
+}
+
+.list-heading,
+.main-content-heading,
+.rating-by,
+.caption-name,
+.caption-name {
+    font-family: "Bakbak One";
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+}
+
+.subscribe-list ul li a,
```

Comment:
Don’t use more than two levels of selector nesting.
Deeply nested selectors (e.g., .subscribe-list ul li a) can become difficult to maintain and understand. It also increases CSS specificity, which can make it harder to override styles later. Keeping your selectors shallow improves readability and maintainability.